src/redux/reducers/updateNum.js

```diff
@@ -0,0 +1,11 @@
+const initialState = 0;
+
+const UpdateNUm = (state = initialState, action) => {
```

Comment:
can you fix the typo here? it should be `UpdateNum`

Reply:
resolved